qml/About.qml

```diff
@@ -24,10 +24,6 @@ Page {
          anchors.centerIn: parent
          spacing: units.gu(4)
          width: parent.width-spacing
-         // #335280 is the new blue by Canonical
-         // https://github.com/CanonicalLtd/desktop-design/blob/master/Colour/colour.png
-         property string themableBlue: Theme.name == "Ubuntu.Components.Themes.SuruDark" ? UbuntuColors.blue : "#335280"
-         property string linkColor: " style=\"color:"+about_column.themableBlue+";\""  //' style="color:'+ about_column.themableBlue +';"'
```

Comment:
The `linkColor` property is handy and should be still used in every html string later in the code because it encapsulate the color into the html style attribute, but it needs to point to `theme.palette.normal.activity` instead, like you pointed out. Maybe you just overlooked this 😅  but thanks for the PR, this color was not part of the theme palette yet when has been introduced here

Reply:
Haha, yes, you are right. I was too much in "cleaning mode". Of course keeping the property and set the color once is much better. I will fix that.